InputFiles/c5g7_mox

```diff
@@ -113,7 +113,7 @@ viz {
 
 nuclearData {
   handles {
-    ce {type aceNeutronDatabase; aceLibrary /home/mak60/myACE/JEF311.aceXS;}
+    ce {type aceNeutronDatabase; aceLibrary $SCONE_ACE;}
   }
   materials {
     numberOfGroups 69;
```

Comment:
Maybe we should delete this line since it isn't used in this input.

Reply:
100% agree (encourage though to use GitHub suggestion feature ;-) )